packages/hash/frontend/src/components/hooks/useUpdateAuthenticatedUser.ts

```diff
@@ -105,7 +105,7 @@ export const useUpdateAuthenticatedUser = () => {
         }
 
         const updatedAuthenticatedUser = constructAuthenticatedUser({
-          userEntityId,
+          userEntityEditionId: userEntityId,
```

Comment:
Should we rename the other variable? It's either the `EntityId` or it's the `EntityEditionId` it shouldn't be both

Reply:
I haven't changed the function sorrounding this call yet, this was due to renaming the param in `constructAuthenticatedUser`

Reply:
Hm, maybe it's fine to do in a follow-up then, I just don't want to see us miss this. As this seems like a bug in how I'm reading it at the moment